plugin/builder/algolia/templates/template.php

```diff
@@ -10,10 +10,11 @@
 
 <?= $container($props, $attrs) ?>
     <algolia-search
-        algolia-app-id="<?php echo $config['app_id']; ?>"
-        algolia-search-key="<?php echo $config['search_key']; ?>"
-        algolia-index-name="<?php echo $config['index_name']; ?>"
-        algolia-routing-refinements=<?php echo $node->routing; ?>
+        algolia-app-id="<?= $config['app_id']; ?>"
+        algolia-search-key="<?= $config['search_key']; ?>"
+        algolia-index-name="<?= $config['index_name']; ?>"
+        algolia-routing-refinements=<?= $node->routing; ?>
```

Comment:
mancano le virgolette credo

Reply:
Pensavo pure io, però gli arriva un json con già le virgolette quindi l'ho lasciato così

Reply:
Se le aggiungo va in errore